data Error.*: [] {
}

func ensure: (v: Bool) -> () {
    if !v {
        throw()
    }
}
